lib/exsoda/http.ex

```diff
@@ -136,10 +136,11 @@ defmodule Exsoda.Http do
   end
 
   defp add_opt(opts, user_opts, key, default) do
-    case conf_fallback(user_opts, key) do
-      nil -> Map.put(opts, key, default)
-      value -> Map.put(opts, key, value)
+    r = case conf_fallback(user_opts, key) do
+      nil -> default
+      value -> value
     end
+    Map.put(opts, key, r)
```

Comment:
This was me figuring out why the tests were broken; I left it because I liked the factored-out Map.put better.

Reply:
Agree. It's much clearer what's different like this.